driver/api/impl/info.cpp

```diff
@@ -176,7 +176,10 @@ SQLRETURN SQL_API EXPORTED_FUNCTION_MAYBE_W(SQLGetInfo)(
             CASE_FALLTHROUGH(SQL_CONVERT_TIMESTAMP)
             CASE_FALLTHROUGH(SQL_CONVERT_INTERVAL_YEAR_MONTH)
             CASE_FALLTHROUGH(SQL_CONVERT_INTERVAL_DAY_TIME)
+
+#if defined(SQL_CONVERT_GUID)
             CASE_NUM(SQL_CONVERT_GUID, SQLUINTEGER, mask_common_CONVERT_dest)
```

Comment:
That should probably switch places with some of the upper `CASE_FALLTHROUGH` for them to be terminated properly at `mask_common_CONVERT_dest`.